examples/sample-app/README.md

```diff
@@ -13,7 +13,7 @@ This section covers how to perform all the steps of building, deploying, and upd
 
 All commands assume the `openshift` binary is in your path (normally located under origin/_output/local/go/bin):
 
-1. Pre-pull the docker images used in this sample.  This is not strictly necessary as OpenShift will pull the images as it needs them, but by doing it up front it will prevent lengthy operations during build and deployment which might otherwise lead you to believe the process has failed or hung.
+1. Pre-pull the docker images used in this sample.  This is not strictly necessary as OpenShift will pull the images as it needs them, but by doing it up front it will prevent lengthy operations during build and deployment which might otherwise lead you to believe the process has failed or hung. (IT IS NOT CLEAR WHAT DIRECTORY THIS IS IN)
 
```

Comment:
yeah that is what I mean. When I wrote this note I was not sure, not having done it a few times I would have just made the update.